great_expectations/experimental/datasources/pandas_datasource.py

```diff
@@ -177,14 +189,35 @@ def get_batch_list_from_batch_request(
         return batch_list
 
 
+_ASSET_MODELS = _generate_data_asset_models(
+    _DataFrameAsset,
+    whitelist=(
+        "read_csv",
+        "read_json",
+        "read_excel",
+        "read_parquet",
+    ),
+)
+
+CSVAsset = _ASSET_MODELS["csv"]
+ExcelAsset = _ASSET_MODELS["excel"]
+JSONAsset = _ASSET_MODELS["json"]
+ParquetAsset = _ASSET_MODELS["parquet"]
```

Comment:
Since we no longer have a model with nicely annotated fields to look at.
One thing we could do is commit the generated json schemas to source control. That way we can catch if they deviate in unexpected ways.

You can use `invoke schema csv|excel|json` etc. to view them. But it's hard to see how things are changing.
Also, it would be trivial to write a tests to make sure the schemas under source control are always in sync with the latest model.